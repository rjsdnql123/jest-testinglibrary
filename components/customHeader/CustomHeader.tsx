import React from "react"; 

function CustomHeader({comment}: any) {
    console.log(comment, 'comment')
    return (
        <div>
            {comment}
        </div>
    )
}

export default CustomHeader;