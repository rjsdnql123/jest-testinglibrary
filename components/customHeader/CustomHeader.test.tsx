import { render, screen } from '@testing-library/react'
import CustomHeader from './CustomHeader'

describe('CustomHeader ', () => {
  it('헤더 이름이 잘 뜨는가?', () => {
    const { container } = render(<CustomHeader comment="TDD 배우기"/>)

    const testFirst = screen.getByText(/TDD 배우기/i)


    expect(testFirst).toBeInTheDocument()

    expect(container).toMatchSnapshot()
  })
})