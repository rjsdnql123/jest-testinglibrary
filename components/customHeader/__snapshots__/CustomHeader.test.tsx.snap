// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomHeader  헤더 이름이 잘 뜨는가? 1`] = `
<div>
  <div>
    TDD 배우기
  </div>
</div>
`;
